acceptance/launcher_test.go

```diff
@@ -42,6 +42,28 @@ func TestLauncher(t *testing.T) {
 }
 
 func testLauncher(t *testing.T, when spec.G, it spec.S) {
+	when("Buildpack API >= 0.5", func() {
+		when("exec.d", func() {
+			it.Before(func() {
+				h.SkipIf(t, runtime.GOOS == "windows", "exec.d is not yet supported on windows")
+			})
+
+			it("executes the binaries and modifies env before running profiles", func() {
+				cmd := exec.Command("docker", "run", "--rm",
+					"--env=VAR_FROM_EXEC_D=ORIG_VAL",
+					launchImage, "exec.d-checker")
+				expected := "/layers/0.5_buildpack/some_layer/exec.d/helper was executed\n"
+				expected += "Exec.d Working Dir: /workspace\n"
+				expected += "/layers/0.5_buildpack/some_layer/exec.d/exec.d-checker/helper was executed\n"
+				expected += "Exec.d Working Dir: /workspace\n"
+				expected += "sourced bp profile\n"
+				expected += "sourced app profile\n"
+				expected += "VAR_FROM_EXEC_D: ORIG_VAL:VAL_FROM_EXEC_D:VAL_FROM_EXEC_D"
```

Comment:
Why is `VAL_FROM_EXEC_D` printed twice?

Reply:
The first `VAL_FROM_EXEC_D` comes from `exec.d/helper` and the second comes from `exec.d/<process-type>/helper` I am happy to add a comment here or I change the actual exec.d logic to make the appended values different in these two cases. Do you have a preference?

Reply:
Ah, this makes sense. I am partial to having the values be different, seems more durable than a comment.